src/Features/ShippingLabelBanner.php

```diff
@@ -110,7 +110,7 @@ public function add_print_shipping_label_script( $hook ) {
 		wp_enqueue_script(
 			'print-shipping-label-banner',
 			Loader::get_url( 'wp-admin-scripts/print-shipping-label-banner.js' ),
-			array( 'wc-navigation', 'wp-i18n', 'wp-data', 'wp-element', 'moment' ),
+			array( 'wc-navigation', 'wp-i18n', 'wp-data', 'wp-element', 'moment', WC_ADMIN_APP ),
```

Comment:
>I think we would have to re-implement updateOptions if we didn't have this dependency.

Maybe its worth it?

In any case, https://github.com/woocommerce/woocommerce-admin/pull/3313 implements the first of several data stores to a `@woocommerce/data` package, allowing you to do this:

```js
import { withDispatch } from '@wordpress/data';
...
withDispatch( ( dispatch ) => {
    const { updateOptions } = dispatch( OPTIONS_STORE_NAME );
    return { updateOptions };
} )
```
This will be more of a priority after the 4.0 release, so perhaps this temporary solution is good enough until this package is available.

Reply:
#3313 would be great.

If you think this is currently a reasonable temporary solution, I'll go ahead with it, and we'll loop back to use `@woocommerce/data` when it becomes available.